documentation/Migration/Update-0.109.md

```diff
@@ -30,17 +30,19 @@ protected override bool IsFileSupportedCore(string path)
         (Path.GetExtension(fds.FullPath) == ".txt")
  }
  ````
- If you are implementing `ICustomDataSource` directly then you will change you `IsFileSupported(string)` method to `IsDataSourceSupported(IDataSource)` and update the logic similar to above.
+ If you are implementing `ICustomDataSource` (now named `IProcessingSource` - see below) directly then you will change you `IsFileSupported(string)` method to `IsDataSourceSupported(IDataSource)` and update the logic similar to above.
 
  ## Engine
 
 The following are required if you are using the `Engine`:
 
-- Try-catch blocks that were expecting `UnsupportedDataSourceException`s to signal an invalid `CustomDataSource` in any of the `Add*` methods should be updated to catch `UnsupportedCustomDataSourceException`:
+- `UnsupportedCustomDataSourceException` has been renamed to `UnsupportedProcessingSourceException`
+
+- Try-catch blocks that were expecting `UnsupportedDataSourceException`s to signal an invalid `CustomDataSource`, (now named `ProcessingSource` - see below) in any of the `Add*` methods should be updated to catch `UnsupportedProcessingSourceException`:
```

Comment:
I think I need to fix this. For 108 -> 109 it's just `UnsupportedDataSourceException` -> `UnsupportedProcessingSourceException`, since the `UnsupportedDataSourceException` -> `UnsupportedCustomDataSourceException` change was made in a 109 preview already